src/controllers/facebookAuth.ts

```diff
@@ -0,0 +1,70 @@
+import { v1 as uuidv1 } from 'uuid';
+import { cleanseData, getSignedJwtToken } from '../utils';
+import { db } from '../database/db';
+import { UsersRepository } from '../database';
+import { CreateUserSchema } from '../database/models';
+
+class FacebookAuthController {
+    constructor(private readonly usersRepository: UsersRepository) {}
+
+    /**
+     * @desc Login using facebook
+     * @route GET /api/auth/facebook-login
+     * @access  Public
+     */
+    facebookLogin = async (req: any, res: any, next: any) => {
+        const profile = req.user;
```

Comment:
What is the shape of the object `profile`? Could you add the typescript object definition here?